UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
I enable the option just in case, but the warnings doesn't seem to come from here. In fact, I don't even know why I'm getting that warnings since the `trailing_newline` rule is respected in all the files. I tried different solution: fixing myself, `swiftlint autocorrect`... nothing changed. 

I'm gonna try on master to see if I'm getting the same warnings